textile/features.textile

```diff
@@ -2322,7 +2324,8 @@ class Push: RSH1, RSH2
     registerCallback: ((ErrorInfo?, DeviceDetails?) -> io String)?,
     // Only on platforms that, after first set, can update later its push
     // device details:
-    updateFailedCallback: ((ErrorInfo) ->)
+    updatedCallback: ((ErrorInfo) ->)?,
```

Comment:
I think it's more readable with the optional argument at the end. Also, shouldn't it be

```suggestion
    updatedCallback: ((ErrorInfo?) ->)?,
```

Reply:
https://github.com/ably/specification/pull/160/commits/0699da3656e120c524af9c0f76b08957f7da3525